### ISSUE TITLE
‘Execute Workflow‘ node always return error `JSON parameter need to be an valid JSON`

### ISSUE THREAD

**Issue by stammi922** [n8n-io/n8n#5326]
**Describe the bug**
In my self-hosted (docker) n8n installation the "Execute Workflow" node is broken. 
My current test workflow includes a manual `When clicking "Execute Workflow"` node connected to a `Set` node that is connected to a `Execute Workflow` node. 
Upon launching the workflow I am receiving the error message 
```
Problem in node ‘Execute Workflow‘
JSON parameter need to be an valid JSON
```

I have also tried connecting the a code node that return a static JSON Object Instead, but I am running into the same error message. 

![image](https://user-images.githubusercontent.com/2290801/216107578-18ca7c58-428c-4685-b098-5ef79daee553.png)

The receiving end is a simple flow that is triggered as a `Execute Workflow Trigger`.

![image](https://user-images.githubusercontent.com/2290801/216107796-d281e668-6b03-4e2b-86b3-2d64f9a3502e.png)

P.S. I of cause made sure that the workflow ID in the Execute node is triggering the right trigger 

**To Reproduce**

1. Go to 'Execute Workflow'
2. See error

**Expected behavior**

`Execute Workflow` node triggers the `Execute Workflow Trigger`

**Environment (please complete the following information):**

- OS:  Ubuntu 22.04.1 LTS (GNU/Linux 5.15.0-1028-aws x86_64)
- n8n Version: 0.213.0
- Database system: SQLite
- Operation mode: own

**Additional context**

**Comment by Joffcom** [comment]
Hey @stammi922,

Can you share the JSON for the workflows so we can try and reproduce the issue?

**Comment by janober** [comment]
Just tested, the node still seems to work fine.

The error you see in the "calling workflow" comes from the "receiving workflow". It simply gets propagated up. So one of the nodes seems to throw that error. So you should be able to debug the "receiving workflow".

**Comment by stammi922** [comment]
Thanks for the quick replies and advice. 

@Joffcom, I'll share both the sending and receiving workflows here:
[Archive.zip](https://github.com/n8n-io/n8n/files/10567301/Archive.zip)

@janober also tried to simplify the workflow with just a `Set` block and it worked. It seems to be having an issue with the `http` block right after the receiving block.

**Comment by Joffcom** [comment]
Hey @stammi922,

That sounds like it is not an issue with n8n and more of a workflow issue which would be better suited to our [community support forum](https://community.n8n.io).

To get your example working I have updated the JSON being sent to...

```
{
    "model": "text-davinci-003",
    "prompt": "Translate the following job title to english: {{  $json.title}}",
    "temperature": 0,
    "max_tokens": 37
}
```

This then works are expected, One thing I have noticed though is you are using the HTTP Request node to make a call to OpenAI. Have you thought about using the node we have for it which would allow the same functionality?
